specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -17190,6 +17214,30 @@
           "items": {
             "$ref": "#/definitions/IPConfig"
           }
+        },
+        "recoveryNicName": {
+          "description": "The name of the NIC to be used when creating target NICs.",
+          "type": "string"
+        },
+        "recoveryNicResourceGroupName": {
+          "description": "The resource group of the NIC to be used when creating target NICs.",
+          "type": "string"
+        },
+        "reuseExistingNic": {
+          "description": "A value indicating whether an existing NIC is allowed to be reused during failover subject to availability.",
+          "type": "boolean"
+        },
+        "tfoNicName": {
+          "description": "The name of the NIC to be used when creating target NICs in TFO.",
+          "type": "string"
+        },
+        "tfoNicResourceGroupName": {
+          "description": "The resource group of the NIC to be used when creating target NICs in TFO.",
+          "type": "string"
+        },
+        "tfoReuseExistingNic": {
+          "description": "A value indicating whether an existing NIC is allowed to be reused during test failover subject to availability.",
+          "type": "boolean"
```

Comment:
adding new properties to existing api versions is breaking change .
Older clients will break due to the new additional properties. 
please introduce these in new api version..

See 'New property added to response':
[https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo)

Reply:
This will not be a breaking change. The property is optional and the use case is not for multiple clients. As per the guidelines, GET-PUT pipeline would be broken when there are multiple clients accessing the properties. THat will not be the case here.